app/src/main/java/com/teamltt/carcare/DemoActivity.java

```diff
@@ -59,12 +59,91 @@ public void onReceive(Context context, Intent intent) {
     private boolean connecting = false;
     // True while the app has a connected socket to the BT device
     private boolean connected = false;
+    // UUID that is required to talk to the OBD II adapter
     private UUID uuid = new UUID(0L, 0x1101L);
```

Comment:
See the other UUID comment. This 0x1101 number isn't what a UUID should look like, it was while I was still figuring things out.